<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoEditDialog</class>
 <widget class="QDialog" name="TodoEditDialog">
  <layout class="QGridLayout" name="dialogLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="editArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum"/>
     </property>
     <widget class="QWidget" name="editContents">
      <layout class="QVBoxLayout" name="editLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="summaryEdit"/>
       </item>
       <item>
        <widget class="QCheckBox" name="doneBox">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="dateButton">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QMaemo5ValueButton" name="calendarButton">
         <property name="text">
          <string>Calendar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="alarmBox">
         <property name="text">
          <string>Alarm</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="alarmLayout">
         <item>
          <widget class="QMaemo5ValueButton" name="alarmDateButton">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Alarm</string>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>2</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QMaemo5ValueButton" name="alarmTimeButton">
           <property name="visible">
            <bool>false</bool>
           </property>
           <property name="valueLayout">
            <enum>QMaemo5ValueButton::ValueBesideText</enum>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ExpandingTextEdit" name="descriptionEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <enum>QDialogButtonBox::Cancel</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ExpandingTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
